### DISCUSSION TITLE
孙吧的帖子，来围观围观

### DISCUSSION THREAD

**Discussion by aiTunes-dev** [share121/inter-knot#1637] [原神]
**一个感人的故事**
<a href="https://tieba.baidu.com/p/9101254039?pn=1">传送门</a>

**Comment by nzj2004** [comment]
现在的贴吧怎么网页直接访问啊？不想下APP

**Reply by ynsuchdle** [reply]
可以试试这个脚本
https://github.com/WhiteSevs/TamperMonkeyScript/tree/master/scripts/【移动端】百度系优化/

**Comment by yui0530** [comment]
在某乎看过截图，怎么说呢，事可能是真事，但里面编的成分很大
